documents/postman/README.md

```diff
@@ -0,0 +1,59 @@
+# Postman
+
+To download Postman, go [here](https://www.postman.com/downloads/). Postman allows you to send requests.
+
+## Postman collection
+
+The intention of this collection is to provide quick, easy functionality to send email, sms, and push notifications.  This collection can be used by business line users integrating with VA Notify to get familiar with the API and send test notifications.  
+
+The postman scripts use the environment variables and populate or update them as the scripts are executed.
+
+Creation, viewing and editing of templates can be done in the Self Service Portal.
```

Comment:
@edDocMe360 below this section can we mention what they should do to get started? E.g. import the collection and env? If I'm following this readme as a client and stepping through it that's where I see a gap.

Reply:
Done. Maybe read and make sure you like the voice?

Reply:
Beautiful!